test-app/.eslintrc.js

```diff
@@ -77,13 +97,5 @@ module.exports = {
       },
       extends: ['plugin:n/recommended'],
     },
-    {
-      // test files
-      files: ['tests/**/*-test.{js,ts}'],
-      extends: [
-        // @todo enable this while fixing linting errors in tests...
-        // 'plugin:qunit/recommended'
```

Comment:
We should keep track that we should enable `plugin:qunit/recommended` for the tests at some point of time. Do you mind moving those lines into the two new blocks for test files?